web_app.py

```
@@ -5239,16 +5239,18 @@ async def generate() -> AsyncIterator[str]:
 
 
 class ResolveDecisionsRequest(BaseModel):
-    # 皇帝亲裁结果：按决策点 idx 顺序，每项 {label, hint?, note?}；
-    # dossier 批红另须带 dossier_id/dossier_decision（#1490，非法不落 decided）。
+    # #1589：皇帝亲裁结果，每项须显式携带 decision_key（{decision_key, label, hint?, note?}）；
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)